wordpress-mobile/WordPress-iOS
# Updates WordPressUI version.

Integrates some changes from WordPressUI to fix an issue on iPad.

WordPressUI PR: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/94

This is a rather important PR since it's holding me back from merging: https://github.com/wordpress-mobile/WordPress-iOS/pull/16649

## To test:

Perform the same test in `develop` to see the difference.

- Place a breakpoint in `PrepublishingViewController:449`, right on the method definition for `handleDismiss()`.
- Create a post.
- Tap on publish, and you will see a popup / bottom sheet with publishing options.
- Dismiss it either by dragging it down, or tapping outside of it.
- Make sure the breakpoint is hit in both iPhone and iPad.

## Regression Notes

1. Potential unintended areas of impact

If there's any reason why this code makes sense in iPhone and not in iPad, there could be a regression.  From what I saw in my tests, either the code makes sense for both or it doesn't for either.

Ping @leandroalonso, @yaelirub since you are the last names I can see in `PrepublishingViewController:449`, in case you want to double check my changes.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Spent some time testing the change, and understanding the code.  We're rolling back the state of a post under certain conditions, but I don't see anything that hints it to be platform-specific.

3. What automated tests I added (or what prevented me from doing so)

None, as it's not really testable.

PR submission checklist:
